packages/visualizations/app/styles/navi-visualizations/visualizations/table.less

```diff
@@ -74,18 +85,17 @@
       cursor: grabbing;
     }
 
-    &.custom-name {
-      text-decoration: underline;
-      text-decoration-style: dotted;
-      text-decoration-color: #aaaaaa;
-    }
-
     &__icon {
       padding: 3px 5px;
     }
 
     &__title {
       flex: 1;
+      &--custom-name {
+        text-decoration: underline;
+        text-decoration-style: dotted;
+        text-decoration-color: #aaaaaa;
```

Comment:
Looks like this was just moved, but are none of our exisiting grays similar to this?